src/ServerResolver_nosrv.cpp

```diff
@@ -18,11 +18,13 @@ class ServerResolverPrivate : public QObject {
 
 		void resolve(QString hostname, quint16 port);
 		QList<ServerResolverRecord> records();
+		ServerResolverError lastError();
```

Comment:
Why a public getter when the field is public itself?

Reply:
This is the case for `m_resolved` too. It's probably an error, I would expect it to be private.